pass-model/src/main/java/org/dataconservancy/pass/model/Repository.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2018 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataconservancy.pass.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Describes a Repository. A Repository is the target of a Deposit.
+ * @author Karen Hanson
+ * @version $Id$
+ */
+
+public class Repository extends PassEntity {
+
+    @JsonProperty("@type")
+    protected String type = PassEntityType.REPOSITORY.getName();
+    
+    /** Name of repository e.g. "PubMed Central" */
+    private String name;
+
+    /** Several sentence description of repository */
+    private String description;
+
+    /** URL to the homepage of the repository so that PASS users can view the platform before deciding whether to participate in it */
+    private String url;
+
+    
+    @Override
+    public String getType() {

```

Comment:
Does `Repository` need to override `getType()` from `PassEntity`?

Reply:
Why not?